projects/snyk.io/package.yml

```diff
@@ -0,0 +1,33 @@
+distributable:
+  url: https://registry.npmjs.org/snyk/-/snyk-{{version}}.tgz
+  strip-components: 1
+
+versions:
+  npm: snyk
+
+dependencies:
+  nodejs.org: ^20
```

Comment:
Probably it supports more versions than this. Not sure how important that is.

Reply:
Ideally we'd make the main binary a shim that adds the right node thus not imposing it on the user’s devenv, but that's not something I've made easy yet.